pkg/dedup/iter.go

```diff
@@ -23,6 +23,12 @@ import (
 // that timestamps are in milliseconds and sampling frequencies typically multiple seconds long.
 const initialPenalty = 5000
 
+const (
```

Comment:
Is there a wiki or comment I can tell these three dedup different for handling series?

Reply:
it is in command flag, the penalty one is original thanos OSS implementation, the second one is prometheus, quorum is what we added to fix a number of issues we see during dedup.